k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -3648,6 +3649,27 @@ public void run() {
 
     }
 
+    @SuppressLint("NewApi") // used for debugging only
+    public void debugClearMessagesLocally(final List<LocalMessage> messages) {
+        if (!K9.DEBUG) {
```

Comment:
Despite the looks `K9.DEBUG` is not constant and tied to the "debug logging" setting. Please use `BuildConfig.DEBUG`.